ibm/data_source_ibm_dl_gateway.go

```diff
@@ -73,6 +73,11 @@ func dataSourceIBMDLGateway() *schema.Resource {
 				},
 			},
 
+			dlAuthenticationKey: {
+				Type:        schema.TypeString,
+				Computed:    true,
```

Comment:
I think its a sensitive value can you mark it as 
sensitive: true, if its a senstive value?

Reply:
This authentication key displays the CRN of the key rather than the key secret. So it is okay to show this information